scripts/src/custard.ts

```diff
@@ -65,19 +65,39 @@ export type Config = {
   // Pattern to ignore filenames or directories.
   ignore?: string | string[];
 
-  // Commands like `custard run <command> [args]...`
+  // Commands like `custard run <config-file> <command> [args]...`
   commands?: {[k: string]: Command};
 
   // Packages to always exclude.
   'exclude-packages'?: string | string[];
 };
 
-/**
- * @param flags command line flags
- * @returns usage string
- */
-function usage(flags: string): string {
-  return `usage: node custard.ts ${flags}`;
+switch (process.env.CUSTARD_VERBOSE || 'info') {
```

Comment:
How is this variable set? Do you need to add it to the ci-setup.json?

Reply:
No, it's not intended to use for contributors. It's more for us to debug / develop the tool if we need extra logging levels. So it's just setting it locally like `export CUSTARD_VERBOSE=debug`.